tools/unix/clang-format.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "Running clang-format on all repository files..."
+
+XARGS_COMMAND='xargs -n1 -0 -P0 clang-format -i'
+
+# Android
+find android/{app,sdk}/src -type f -name '*.java' -print0 | $XARGS_COMMAND
+find android/sdk/src/main/cpp -type f -name '*.[hc]pp' -print0 | $XARGS_COMMAND
```

Comment:
Still don't like it. Can you combine those lines into one to make it simpler?
I won't mind if `find` will look for cpp sources in `android/app/src`